rust/lance-index/src/scalar/inverted/query.rs

```diff
@@ -358,28 +392,25 @@ impl MultiMatchQuery {
         Ok(Self { match_queries })
     }
 
-    pub fn try_new_with_boosts(
-        query: String,
-        columns: Vec<String>,
-        boosts: Vec<f32>,
-    ) -> Result<Self> {
-        if boosts.len() != columns.len() {
+    pub fn try_with_boosts(mut self, boosts: Vec<f32>) -> Result<Self> {
```

Comment:
This was a breaking change, but the PR wasn't labelled as a breaking change. Try to make sure to label your PRs as breaking, or else avoid breaking changes in the future.